src/Login/ProtectedContent.jsx

```diff
@@ -83,7 +86,8 @@ class ProtectedContent extends React.Component {
             },
           ),
       );
-    if (this.props.public) {
+    // if userAccessToSite is enabled do not load homepage until access is checked
+    if (this.props.public && !(this.props.match.path === '/' && userAccessToSite?.enabled)) {
```

Comment:
why check the path here? shouldn't it block access to any public pages if this feature is enabled?

Reply:
good catch, this was done because this needs to run on the login page. Will reverse the logic to look for that page instead and update comment